enterprise/api/v1/serializers.py

```diff
@@ -562,6 +571,8 @@ def create(self, validated_data):
                 else:
                     enterprise_customer_user.unenroll(course_run_id)
             except (CourseEnrollmentDowngradeError, CourseEnrollmentPermissionError, HttpClientError) as exc:
+                error_message = '[Enterprise API] An exception occurred {}.'.format(str(exc))
```

Comment:
i would change this to: "An exception occurred while enrolling the user..."

Reply:
Done